spark/src/main/scala/org/allenai/pipeline/spark/PartitionedRddFileArtifact.scala

```diff
@@ -0,0 +1,42 @@
+package org.allenai.pipeline.spark
```

Comment:
This Artifact class is a common interface for saving RDDs.  Previously, we had separate implementations for File- and S3-based artifacts, which required some ugly run-time dynamic casting.